virtual-server/examples/kubectl/virtual-server-shared-pvc.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: your-pv-claim
+spec:
+  accessModes:
+  - ReadWriteMany
+  storageClassName: shared-nvme-ewr1
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 20Gi
+---
+apiVersion: virtualservers.coreweave.com/v1alpha1
+kind: VirtualServer
+metadata:
+  name: example-vs-shared-pvc
+  namespace: my-namespace
+spec:
+  region: ORD1
+  os:
+    type: linux
+  resources:
+    gpu:
+      type: Quadro_RTX_4000
+      count: 1
+    cpu:
+      count: 3
+    memory: 16Gi
+  storage:
+    root:
+      size: 40Gi
+      storageClassName: block-nvme-ord1
+      source:
+        pvc:
+          namespace: vd-images
+          name: ubuntu2004-docker-master-20210323-ord1
+    sharedFs:
```

Comment:
I don't think this is the right syntax for VS. did you test?

Reply:
I hadn't meant to commit that file yet (because it hadn't worked-- I think I was basing this on an old template from kubeapps I had looked at for the TF example), but found what I needed, though this sample will need to be updated as well: https://github.com/coreweave/virtual-server-operator/blob/c30b0fcc3e30771edb2763ba9bfd71e0ecabec90/config/samples/_v1alpha1_virtualserver.yaml#L25-L33 because filesystems and additionalDisks (according to docs, and in testing) should be defined the same way